catchup/catchpointService.go

```diff
@@ -156,11 +160,16 @@ func MakeNewCatchpointCatchupService(catchpoint string, node CatchpointCatchupNo
 }
 
 // Start starts the catchpoint catchup service ( continue in the process )
-func (cs *CatchpointCatchupService) Start(ctx context.Context) {
+func (cs *CatchpointCatchupService) Start(ctx context.Context) error {
+	err := cs.checkLedgerDownload()
+	if err != nil {
+		return err
+	}
```

Comment:
Add some extra information about what we're doing as a result of the error. An error in `checkLedgerDownload` could also be helpful.

```suggestion
	err := cs.checkLedgerDownload()
	if err != nil {
		return fmt.Errorf("Start(): aborting catchup: %s", err)
	}
```